tests/test_cast_to_type.py

```diff
@@ -10,15 +10,18 @@
 # limitations under the License.
 
 import unittest
+import torch
 import numpy as np
 from parameterized import parameterized
 from monai.transforms import CastToType
 
 TEST_CASE_1 = [{"dtype": np.float64}, np.array([[0, 1], [1, 2]], dtype=np.float32), np.float64]
 
+TEST_CASE_2 = [{"dtype": torch.float64}, torch.tensor([[0, 1], [1, 2]], dtype=torch.float32), torch.float64]
+
 
 class TestCastToType(unittest.TestCase):
-    @parameterized.expand([TEST_CASE_1])
+    @parameterized.expand([TEST_CASE_1, TEST_CASE_1])
```

Comment:
`TEST_CASE_2` here?

Reply:
Sorry for my typo, fixed in the latest commit.
Thanks.